scitt-components/detached-artifacts.md

```diff
@@ -0,0 +1,8 @@
+# Detached Artifacts
+
+> TODO: Describe the importance for detached signatures, claims and evidence
+
+- Protection from Trojan Horse Scenarios
+- Ability to continually add information to sealed artifacts and evidence
+- Ability to filter what may/should be promoted across an ecosystem
+- Ability to link to existing artifacts by unique IDs and hashes
```

Comment:
We keep talking about "linking to hashes"... but thats only possible if you have a standard hash format.

I suggest we start thinking about an informative example of one.

Reply:
This is a perfect example where we should link to a page/issue/i-d that frames the problem and opens it up for design discussion. This way the doc frames the scenario, with a pointer to an eventual spec.

Reply:
I've added:
> (NOTE: to track how we'll establish unique IDs and/or hashes)